src/ape/utils.py

```diff
@@ -156,6 +215,13 @@ def load_config(path: Path, expand_envars=True, must_exist=False) -> Dict:
 
 
 def compute_checksum(source: bytes, algorithm: str = "md5") -> str:
+    """
+    Compute the md5 checksum against bytes for a string.
```

Comment:
You can specify other algorithms

```suggestion
    Compute the checksum for the given bytes.
```